__tests__/TimelineTest.js

```diff
@@ -36,8 +36,7 @@ describe('Timeline', () => {
 
 	it("creates pending TimelineResultComp", function() {
 		wrapper.setState({timeline: null, error: null});
-		expect(wrapper.containsMatchingElement(React.createElement(
-			TimelineResultComp, {className: "pending"}, null))).toEqual(true);
+		expect(wrapper.containsMatchingElement(React.createElement(TimelineResultComp, {className: 'pending'}, null))).toEqual(true);
 	})
```

Comment:
quotes, also in the future perhaps consider implementing a helper to make the tests less verbose since you'll probably be adding more with each react feature